app/views/govuk_publishing_components/components/attachment/_thumbnail_document.svg

```diff
@@ -0,0 +1,4 @@
+<svg class="gem-c-attachment__thumbnail-image gem-c-attachment__thumbnail-image--svg" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 99 140">
```

Comment:
I think you could squeeze some bytes out of these using a tool like https://jakearchibald.github.io/svgomg/ :)

Reply:
I would also recommend having fallback `height` and `width`, since if CSS fails you don't want these images filling the page.

You could also consider putting `fill=currentColor` onto the path for similar reasons.

Reply:
Thanks! I already gave the a spin before they went up (well I used the node package for it).

Reply:
Fair, I can see some stuff that doesnt need to be there so worth going over it again I think.

Reply:
Hmm you're right. I did save some more again. Kinda weird the tool doesn't get it to the lowest the first time 🤔 

based on an original:

```
➜  tmp npx svgo pub-cover-spreadsheet.svg

pub-cover-spreadsheet.svg:
Done in 34 ms!
0.924 KiB - 42.7% = 0.529 KiB
➜  tmp npx svgo pub-cover-spreadsheet.svg

pub-cover-spreadsheet.svg:
Done in 33 ms!
0.529 KiB - 3.9% = 0.509 KiB
➜  tmp npx svgo pub-cover-spreadsheet.svg

pub-cover-spreadsheet.svg:
Done in 29 ms!
0.509 KiB - 0% = 0.509 KiB
➜  tmp npx svgo pub-cover-spreadsheet.svg
```

Reply:
I'm mostly hoping @alex-ju will take over on SVG front though to get it all nice

Reply:
![Janitor dancing](https://media.giphy.com/media/DZu3mnCcmsJKE/giphy.gif)

Reply:
Updated SVGs to clean up markup and apply styling via CSS

Reply:
Thanks Alex - they look good!

I notice you removed the svg modifier class, so I just wanted to check that these would still display ok if this class was put on an img element? Just for the scenario that we continued to support pdf images? (mostly asking since it's hard to add classes later once embedded in documents)

Reply:
Yes, they will be fine regardless of using an image or an svg - the SVG-specific attributes (`fill` and `stroke`) won't affect a regular bitmap image.

Reply:
Awesome 👍